packages/http-client-csharp/generator/TestProjects/CadlRanch.Tests/Http/Parameters/Spread/SpreadTests.cs

```diff
@@ -1,176 +1,176 @@
-            Assert.AreEqual(204, result.GetRawResponse().Status);
-        });
-
-        [CadlRanchTest]
-        public Task Alias_SpreadWithModel() => Test(async (host) =>
-        {
-            ClientResult result = await new SpreadClient(host, null).GetAliasClient().SpreadParameterWithInnerModelAsync("1", "bar", "foo");
-            Assert.AreEqual(204, result.GetRawResponse().Status);
-        });
-
-        [CadlRanchTest]
-        public Task Alias_SpreadAliasinAlias() => Test(async (host) =>
-        {
-            ClientResult result = await new SpreadClient(host, null).GetAliasClient().SpreadParameterWithInnerAliasAsync("1", "bar", "foo", 1);
-            Assert.AreEqual(204, result.GetRawResponse().Status);
-        });
-
-        private static void ValidateConvenienceMethod(Type clientType, string methodName, IEnumerable<(Type ParameterType, string Name, bool IsRequired)> expected)
-        {
-            var methods = FindMethods(clientType, methodName);
-
-            foreach (var method in methods)
-            {
-                ValidateConvenienceMethodParameters(method, expected);
-            }
-        }
-
-        private static IEnumerable<MethodInfo> FindMethods(Type clientType, string methodName)
-        {
-            var asyncMethodName = $"{methodName}Async";
-            var methods = clientType.GetMethods();
-
-            return methods.Where(m => m.Name.Equals(methodName) || m.Name.Equals(asyncMethodName));
-        }
-
-        private static void ValidateConvenienceMethodParameters(MethodInfo method, IEnumerable<(Type ParameterType, string Name, bool IsRequired)> expected)
-        {
-            if (IsProtocolMethod(method))
-                return;
-
-            expected = expected.Append((typeof(CancellationToken), "cancellationToken", false));
-
-            var parameters = method.GetParameters().Where(p => !p.ParameterType.Equals(typeof(RequestOptions)));
-            var parameterTypes = parameters.Select(p => p.ParameterType);
-            var parameterNames = parameters.Select(p => p.Name);
-            var parameterRequiredness = parameters.Select(p => !p.IsOptional);
-            var expectedTypes = expected.Select(p => p.ParameterType);
-            var expectedNames = expected.Select(p => p.Name);
-            var expectedRequiredness = expected.Select(p => p.IsRequired);
-
-            CollectionAssert.AreEqual(expectedTypes, parameterTypes);
-            CollectionAssert.AreEqual(expectedNames, parameterNames);
-            CollectionAssert.AreEqual(expectedRequiredness, parameterRequiredness);
-        }
-
-        private static bool IsProtocolMethod(MethodInfo method)
-            => method.GetParameters().Any(parameter => parameter.ParameterType.Equals(typeof(BinaryContent)));
-    }
+    // public class SpreadTests : CadlRanchTestBase
```

Comment:
Can you clarify why these are commented out?

Reply:
Here is an issue for this: https://github.com/microsoft/typespec/issues/5565

Reply:
this has been restored.